app/helpers/application_helper/button/pdf.rb

```diff
@@ -2,4 +2,8 @@ class ApplicationHelper::Button::Pdf < ApplicationHelper::Button::Basic
   def visible?
     PdfGenerator.available?
   end
+
+  def check_rbac?
+    %w(render_report_pdf chargeback_download_pdf compare_download_pdf).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
yep that sounds good to me to set ```hidden: true``` inside ```miq_product_features.yml``` for those new defined features according to https://github.com/ManageIQ/manageiq/blob/master/lib/rbac/authorizer.rb#L45..L53